### PR TITLE
Add Detailed Download Steps to getit

### PR DESCRIPTION
**Pull request by mikevallano**

### CODE REVIEW

**Comment by ilario** [comment]
+1 for merging the download tutorial, but regarding the flashing instructions we should discuss if we want them here or in the QuickStartingGuide and just a link to it here. @G10h4ck @p4u @altergui @nicoechaniz

**Comment by G10h4ck** [comment]
I agree with p4u, both +ifconfig+ and +route+ are deprecated since years, use +ip route+ instead

**Comment by mikevallano** [comment]
Thanks for the comments and links. I made the requested edits and added a couple links to resources that go into detail on how to find a router's ip.

File: getit.txt [hunk 1]
```diff
@@ -8,10 +8,23 @@ Get it
...
+
+* If you are installing for the first time, choose the link ending with `-factory.bin`. If you are upgrading an existing install, choose the link ending with `-sysupgrade.bin`.
+
+* Once downloaded, connect your router via ethernet and log into your router's admin. First, you will need to identify the IP address for the router. You can go to the terminal use `netstat -rn` (mac), or `ip route show default` (linux), then find the IP address for the router. More details on finding your router's ip can be found http://www.howtogeek.com/233952/how-to-find-your-routers-ip-address-on-any-computer-smartphone-or-tablet/[here] and http://www.computerworld.com/article/2474776/wireless-networking/network-security-find-the-ip-address-of-your-home-router.html[here]. Copy the IP and then replace the last set with 1. For example for `10.138.21.216`, put `10.138.21.1` in the browser. You can also try replacing the previous set with a zero, so `10.138.21.216` would become `10.138.0.1`.
+
+* Once the IP is found, log in to the admin area for the router. Find the `Flash new firmware image` option.
```
Comment (p4u) [inline getit.txt]: I would say here somehting like "log into the router administration interface by accessing to the router IP (the one obtained previously) via web browser such as Firefox or Chromium: http://<your_router_ip>/"

File: getit.txt [hunk 2]
```diff
@@ -8,10 +8,23 @@ Get it
...
+
+* If you are installing for the first time, choose the link ending with `-factory.bin`. If you are upgrading an existing install, choose the link ending with `-sysupgrade.bin`.
+
+* Once downloaded, connect your router via ethernet and log into your router's admin. First, you will need to identify the IP address for the router. You can go to the terminal use `netstat -rn` (mac), or `ip route show default` (linux), then find the IP address for the router. More details on finding your router's ip can be found http://www.howtogeek.com/233952/how-to-find-your-routers-ip-address-on-any-computer-smartphone-or-tablet/[here] and http://www.computerworld.com/article/2474776/wireless-networking/network-security-find-the-ip-address-of-your-home-router.html[here]. Copy the IP and then replace the last set with 1. For example for `10.138.21.216`, put `10.138.21.1` in the browser. You can also try replacing the previous set with a zero, so `10.138.21.216` would become `10.138.0.1`.
```
Comment (p4u) [inline getit.txt]: I would not say "put `10.138.21.1` in the browser.", It is already explained in the next paragraph.

**Comment by mikevallano** [comment]
Thanks for the feedback. I reworded the paragraph based on the last comments and I think it reads better now.

**Comment by ilario** [comment]
I would move this section to Quick Starting Guide and leave in Get It page just an indication "See installation procedure in Quick Starting Guide". Is it ok, may I proceed? @p4u @mikevallano

**Comment by p4u** [comment]
Yes, I think it is a good idea.

**Pull request merged**